pkg/config/toml/docs.toml

```diff
@@ -535,7 +535,7 @@ Order = 100 # Default
 # IsLoadBalancedRPC indicates whether the http/ws url above has multiple rpc's behind it.
 # If true, we should try reconnecting to the node even when its the only node in the Nodes list.
 # If false and its the only node in the nodes list, we will mark it alive even when its out of sync, because it might still be able to send txs.
-IsLoadBalancedRPC = false
+IsLoadBalancedRPC = false # Example
```

Comment:
This was missing, but since the test for the docs was not running, it was not caught